projects/wave-core/src/lib/operators/dialogs/data-repository/vector/vector-source-dataset.component.ts

```diff
@@ -25,7 +25,7 @@ import {NotificationService} from '../../../../notification.service';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 
-export class VectorSourceDatasetComponent implements OnInit {
+export class VectorSourceDatasetComponent implements OnInit , OnChanges  {
```

Comment:
Please format the code.

Reply:
formated in 6da1d4c74b7a75a702074ebf5ef83c4bda092d4f****